packages/medusa/src/commands/migrate.js

```diff
@@ -1,38 +1,56 @@
 import { createConnection } from "typeorm"
-import { getConfigFile } from "medusa-core-utils"
-
+import configLoader from "../loaders/config"
 import Logger from "../loaders/logger"
 
 import getMigrations from "./utils/get-migrations"
 
-const t = async function({ directory }) {
+const t = async function ({ directory }) {
   const args = process.argv
   args.shift()
   args.shift()
   args.shift()
+  const configModule = await configLoader(directory)
+  const migrationDirs = await Promise.resolve(getMigrations(directory))
+  let hostConfig = {
+    database: configModule.projectConfig.database_database,
+    url: configModule.projectConfig.database_url,
+  }
 
-  const { configModule } = getConfigFile(directory, `medusa-config`)
-  const migrationDirs = getMigrations(directory)
+  if (configModule.projectConfig.database_host) {
+    hostConfig = {
+      host: configModule.projectConfig.database_host,
+      port: configModule.projectConfig.database_port,
+      database: configModule.projectConfig.database_database,
+      ssl: configModule.projectConfig.database_ssl,
+      username: configModule.projectConfig.database_username,
+      password: configModule.projectConfig.database_password,
+    }
+  }
 
-  const connection = await createConnection({
-    type: configModule.projectConfig.database_type,
-    url: configModule.projectConfig.database_url,
-    extra: configModule.projectConfig.database_extra || {},
-    migrations: migrationDirs,
-    logging: true,
-  })
+  let connection = undefined
+  if (configModule) {
```

Comment:
comment: I don't think that configModule should be check here, it should exists since it has been loaded, therefore, all the connection check below should be remove to

Reply:
done